config/routes.rb

```diff
@@ -1,14 +1,17 @@
 Rails.application.routes.draw do
 
-  get 'cart_items/create'
-
+  # get 'tenants/index'
+  #
+  # get 'tenants/show'
+  #
   root 'welcome#index'
-
   post '/login', to: 'sessions#create'
-
   delete '/logout', to: 'sessions#destroy', as: 'logout'
 
-  resources :categories
+  get 'cart_items/create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.